src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -771,6 +771,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Nothing)
         End Function
 
+        Protected Overrides ReadOnly Property CommonScriptGlobalsType As ITypeSymbol
+            Get
+                Throw New NotImplementedException()
```

Comment:
>Throw New NotImplementedException() [](start = 16, length = 35)

```Return Nothing```? If this is not applicable to VB. If applicable, perhaps we should implement it.

Reply:
It is applicable in theory but we have not been adding more VB scripting support.